src/azure-cli/azure/cli/command_modules/network/tests/latest/test_private_endpoint_commands.py

```diff
@@ -447,7 +447,7 @@ def test_private_link_resource_acr(self):
         self.cmd('network private-link-resource list --id {registry_id}', checks=[
             self.check('length(@)', 1)])
 
-    @ResourceGroupPreparer(location='centraluseuap')
+    @ResourceGroupPreparer(location='centralus')
```

Comment:
Just out of curiosity, why do you modify this location for the network module related tests?

Reply:
Because it said that centraluseuap was an unsupported region when I ran the test